cc_licenses/templates/includes/deed_licenses_body.html

```diff
@@ -0,0 +1,168 @@
+
+  {% if legalcode|is_one_of:"devnations" %}
+
+      <p class="has-text-black body-big padding-bottom-normal">{% blocktrans %}<strong>to Share</strong> &mdash; to copy, distribute and transmit the work{% endblocktrans %}</p>
+      <p class="has-text-black body-big padding-bottom-normal">{% blocktrans %}<strong>to Remix</strong> &mdash; to adapt the work{% endblocktrans %}</p>
+
+  {% elif legalcode|is_one_of:"nc-sampling+,sampling,sampling+" %}
+
+    {% if license.prohibits_commercial_use %}
+      <p class="has-text-black body-big padding-bottom-normal">{% blocktrans %}To sample, mash-up, or otherwise creatively transform this work for noncommercial purposes.{% endblocktrans %}</p>
+    {% else %}
+      <p class="has-text-black body-big padding-bottom-normal">{% blocktrans %}To sample, mash-up, or otherwise creatively transform this work for commercial or noncommercial purposes.{% endblocktrans %}</p>
+    {% endif %}
+
+    {% if legalcode|is_one_of:"nc-sampling+,sampling+" %}
+      <p class="has-text-black body-big padding-bottom-normal">{% blocktrans %}To perform, display, and distribute copies of this whole work for noncommercial purposes (e.g., file-sharing or noncommercial webcasting).{% endblocktrans %}</p>
+    {% endif %}
+
+  {% else %}
+
+    {% if license.prohibits_commercial_use %}
+      <p class="has-text-black body-big padding-bottom-normal">{% blocktrans %}<strong>Share</strong> &mdash; copy and redistribute the material in any medium or format{% endblocktrans %}</p>
+    {% else %}
+      <p class="has-text-black body-big padding-bottom-normal">{% blocktrans %}<strong>Share</strong> — copy and redistribute the material in any medium or format for any purpose, even commercially.{% endblocktrans %}</p>
+    {% endif %}
+
+    {% if license.permits_derivative_works %}
+      {% if license.prohibits_commercial_use %}
+        <p class="has-text-black body-big padding-bottom-small">{% blocktrans %}<strong>Adapt</strong> &mdash; remix, transform, and build upon the material{% endblocktrans %}
+        </p>
+      {% else %}
+        <p class="has-text-black body-big padding-bottom-small">{% blocktrans %}<strong>Adapt</strong> &mdash; remix, transform, and build upon the material for any purpose, even commercially.{% endblocktrans %}
+        </p>
+      {% endif %}
+    {% endif %}
+  {% endif %}
+
+  <p class="has-text-black body-big padding-bottom-small">
+    {% blocktrans %}The licensor cannot revoke these freedoms as long as you follow the license terms.{% endblocktrans %}
+  </p>
+
+  {# TERMS #}
+
+  <h3 class="b-header has-text-black padding-bottom-big padding-top-normal" style="font-weight: bold;">
+    {% trans "Under the following terms:" %}
+  </h3>
+  <div class="columns is-multiline">
+
+  {% if license.prohibits_high_income_nation_use %}
+    <div class="column is-1">
+      {# empty div because there is no icon, but we need spacing #}
+    </div>
+    <div class="column is-11">
```

Comment:
Rather than adding empty page elements, there is often some kind of margin utility class, such as:

- [Bulma spacing helpers](https://bulma.io/documentation/helpers/spacing-helpers/)
- [Bootstrap spacing classes](https://getbootstrap.com/docs/4.0/utilities/spacing/)